program/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "paladin-rewards-program"
+version = "0.1.0"
+description = "Paladin Rewards Program"
+readme = "./README.md"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+
+[dependencies]
+bytemuck = "1.16.0"
+solana-program = "1.18.14"
+spl-associated-token-account = { version = "3.0.2", features = ["no-entrypoint"] }
+spl-program-error = "0.4.0"
+spl-tlv-account-resolution = "0.6.3"
+spl-token-2022 = { version = "3.0.2", features = ["no-entrypoint"] }
+spl-transfer-hook-interface = "0.6.3"
+
+[dev-dependencies]
+solana-program-test = "1.18.14"
+solana-sdk = "1.18.14"
+spl-pod = "0.2.2"
+spl-type-length-value = "0.4.3"
+
+[features]
+bpf-entrypoint = []
```

Comment:
The template works under the assumption that the program crate is not published, so I wonder whether we need to specify a feature for the `bpf-entrypoint` or not. How could this be used?